src/System.Private.ServiceModel/src/Resources/Strings.resx

```diff
@@ -6756,8 +6756,16 @@
   <data name="SSLProtocolNegotiationFailed" xml:space="preserve">
     <value>SSL protocol negotiation failed. Requested '{0}' but was offered '{1}'.</value>
   </data>
-  <!-- Copied from System.Net.Primitives net_ssl_io_cert_validation as we're doing the Ssl certificate validation ourselves for UWP apps-->
   <data name="ssl_io_cert_validation" xml:space="preserve">
-    <value>The remote certificate is invalid according to the validation procedure.</value> 
+    <value>The remote certificate is invalid according to the validation procedure.</value>
```

Comment:
Reviewers: this strings were copied verbatim from IdentityModel in the full framework.